jsonschema-module-microprofile-openapi-3/src/main/java/com/github/victools/jsonschema/module/microprofile/openapi3/MicroProfileOpenApi3AnyOfResolver.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 VicTools.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.victools.jsonschema.module.microprofile.openapi3;
+
+import com.fasterxml.classmate.ResolvedType;
+import com.github.victools.jsonschema.generator.SchemaGenerationContext;
+import com.github.victools.jsonschema.generator.SubtypeResolver;
+import com.github.victools.jsonschema.generator.TypeContext;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+/**
+ * Subtype resolver considering {@code @Schema(anyOf = ...)}.
+ */
+public class MicroProfileOpenApi3AnyOfResolver implements SubtypeResolver {
+
+    @Override
+    public List<ResolvedType> findSubtypes(ResolvedType declaredType, SchemaGenerationContext context) {
```

Comment:
I've commented the usage of this class. My Integration tests (only) are passing.
I get it from Swager 2 side.
Do you think that it is useful to reuse it ?

Reply:
Ok so I should remove it.
Is it ok on your side ?

Reply:
Ok I have removed it.

Reply:
Ok so I keep it but removing this bloc

```java
        if (annotation.anyOf().length > 0) {
            ArrayNode allOfArray = memberAttributes.withArray(context.getKeyword(SchemaKeyword.TAG_ALLOF));
            ArrayNode anyOfArray = allOfArray.addObject().withArray(context.getKeyword(SchemaKeyword.TAG_ANYOF));
            Stream.of(annotation.anyOf())
                    .map(context.getTypeContext()::resolve)
                    .map(context::createDefinitionReference)
                    .forEach(anyOfArray::add);
        }
```
will make my tests failing.

I guess I should remove `MicroProfileOpenApi3AnyOfResolver` and keep the previous bloc